### DISCUSSION TITLE
Verify

### DISCUSSION THREAD

**Discussion by IhabSky** [PartialVolume/shredos.x86_64#232] [Ideas]
Hello
first thank you very much for that useful program . wish you add Write and Read verify to let program wipe very useful to secure erase and then add choose ( write verify and  read verify ) to check HDD's to be sure i can trust and use it again with good health  ,without hidden bad block

**Comment by PartialVolume** [comment]
>first thank you very much for that useful program

Thanks, hope it's useful for you.

If your are asking whether it can verify, yes it can already do that,  it can verify zeros, ones post wiping. Under method you will find verify zeros and verify ones or you can verify each pass or last pass at the time of the wipe, zeros, ones and prng.

**Reply by IhabSky** [reply]
Yes I find it and it wonderful when I make it for 10x HDD at same time and also result are great 
Can you help me to add read verify too 
It will be great to let me sure HDD are read and write
